src/idl_gen_grpc.cpp

```diff
@@ -173,45 +176,168 @@ class FlatBufFile : public grpc_generator::File {
           new FlatBufPrinter(str));
   }
 
- private:
+ protected:
   const Parser &parser_;
   const std::string &file_name_;
 };
 
-bool GenerateGRPC(const Parser &parser,
-                  const std::string &/*path*/,
-                  const std::string &file_name) {
 
-  int nservices = 0;
-  for (auto it = parser.services_.vec.begin();
```

Comment:
Again that's taken care by the GenerateGRPC once we call the C++ generator from there.